src/NUnitFramework/tests/Attributes/LifeCycleAttributeTests.cs

```diff
@@ -35,11 +35,11 @@ public class LifeCycleAttributeTests
         [Test]
         public void SetupTearDownIsCalledOnce()
         {
-            var fixture = new SetupAndTearDownFixtureInstancePerTestCase();
-            TestBuilder.RunTestFixture(fixture);
+            var fixture = TestBuilder.MakeFixture(typeof(SetupAndTearDownFixtureInstancePerTestCase));
```

Comment:
See comments above about why I rewrote this test and see other tests in this class.